.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -0,0 +1,43 @@
+---
+name: "\U0001F41B Bug Report"
+about: If something isn't working as expected
+title: ''
+labels: ":construction: Blocked: Accessibility Input, :construction: Blocked: Engineering
+  Input, :construction: Blocked: Functional Input, :construction: Blocked: UX Input"
+assignees: ''
+
+---
+
+# Bug Report
+
```

Comment:
This bug report will refer to any package in the mono repo. As of now we'll have two packages in the mono repo, the terra-graphs package and the carbon-graphs package. Since both of those packages will relate directly to graphs, I think it still makes sense to add this for now.